src/components/interface/SectionHeader.jsx

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import { withStyles } from '@material-ui/core/styles';
+import ArrowLeft from '@material-ui/icons/ArrowLeft';
 import Typography from '@material-ui/core/Typography';
 
+import Button from './Button';
 import styles from '../styles';
 
 const SectionHeader = ({
   classes,
+  history,
   renderNewButton,
   title
-}) => (
-  <div className={classes.sectionHeaderWrapper}>
-    <Typography variant="h3">
-      {`${title}s`}
-    </Typography>
-    {!!renderNewButton && (
-      <div className={classes.sectionHeaderAction}>
-        {renderNewButton}
-      </div>)
-    }
-  </div>
-);
+}) => {
+  const goBack = () => { history.goBack(); };
+  return (
+    <div className={classes.sectionHeaderWrapper}>
+      <div className={classes.sectionHeaderBackButton}>
+        <Button type="default" onClick={goBack}>
```

Comment:
We should also add a back button to the settings page. If we extract the components that make up the back button here and put it into its own component i.e. `BackButton`. then we can use here and it in the Settings Page

Reply:
@YangStephen actually I just realized that Settings is missing a Section Header LOOOL so it's fine we can actually just leave that there and actually just add a section header to settings i.e.
![image](https://user-images.githubusercontent.com/18960825/51093673-ec45be80-175a-11e9-8559-686dc070731b.png)

And then the extraction of the back button is a nice to have but not needed